full-service/src/db/account.rs

```diff
@@ -252,7 +252,9 @@ impl AccountModel for Account {
         use crate::db::schema::accounts;
 
         let account_id = AccountID::from(account_key);
-        let fb = first_block_index.unwrap_or(DEFAULT_FIRST_BLOCK_INDEX);
+
+        let first_block_index = first_block_index.unwrap_or(DEFAULT_FIRST_BLOCK_INDEX);
+        let next_block_index = first_block_index;
```

Comment:
Just noting that this is not a functional change. We explicitly do this to show intent that we mean to set the variables this way.